base_tier_validation/models/tier_validation.py

```diff
@@ -172,6 +171,10 @@ def write(self, vals):
             self.mapped('review_ids').unlink()
         return super(TierValidation, self).write(vals)
 
+    def _check_state_conditions(self, vals):
```

Comment:
```suggestion
    def _check_state_conditions(self, vals):
        self.ensure_one()
```
Just to make sure it is not called in multi.